concreate/cli.py

```diff
@@ -64,12 +64,8 @@ def parse(self):
         build_group.add_argument('--build-tag',
                                  dest='build_tags',
                                  action='append',
-                                 help='tag to assign to the built image, can be used multiple times')
-
-        build_group.add_argument('--tag',
-                                 dest='tags',
-                                 action='append',
-                                 help='tag used to build/test the image, can be used multiple times')
+                                 help='tag to assign to the built image, '
+                                 'can be used multiple times')
```

Comment:
This line breaking is too eager.

Reply:
its not, 101 characters is more than 100 allowed for us

Reply:
This is I don't like it. At all.

Reply:
Can you suggest better breaking?

Reply:
Sure, not breaking it :)

Reply:
I can do something like:
``` python
        build_group.add_argument('--build-tag',
                                 dest='build_tags',
                                 action='append',
                                 help=('tag to assign to the built image, '
                                       'can be used multiple times'))
```

which is maybe indented better?